src/js/tracing/reactnativetracing.ts

```diff
@@ -129,12 +139,30 @@ export class ReactNativeTracing implements Integration {
       // @ts-ignore TODO
       shouldCreateSpanForRequest,
       routingInstrumentation,
+      enableNativeFramesTracking,
     } = this.options;
 
     this._getCurrentHub = getCurrentHub;
 
     void this._instrumentAppStart();
 
+    if (enableNativeFramesTracking) {
+      this.nativeFramesInstrumentation = new NativeFramesInstrumentation(
+        addGlobalEventProcessor,
+        () => {
+          const self = getCurrentHub().getIntegration(ReactNativeTracing);
```

Comment:
Can we just pass a boolean instead of a function?

Reply:
Actually, now that I gave it a second thought shouldn't the check that the integration exists on the hub (and subsequently that it has this native frames instrumentation) be run on every event processor?

Reply:
@AbhiPrasad [All the integrations on the JS repo that set an event processor](https://github.com/getsentry/sentry-javascript/tree/master/packages/integrations/src) seem to check on every event, I assumed it was to support multiple hubs right?

Reply:
Ah good point. Didn't realize this, let's keep the check on every event processor run then.